packages/ssx-gnosis-extension/tests/server.test.ts

```diff
@@ -0,0 +1,46 @@
+import { ethers, Wallet } from "ethers";
+import { SiweError, SiweErrorType, SiweMessage } from "siwe";
+import { SiweGnosisVerify, GnosisDelegation } from "../src";
+
+const provider = new ethers.providers.InfuraProvider(
+  "goerli",
+  "49a0efa3aaee4fd99797bfa94d8ce2f1"
```

Comment:
Is this safe to commit?

Reply:
I believe this is a key for tests, it's also used [here](https://github.com/ethers-io/ethers.js/blob/master/packages/tests/src.ts/test-providers.ts#L289)